thewalrus/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.20.0-dev"
+__version__ = "0.20.1-dev"
```

Comment:
No need to bump the version here!
```suggestion
__version__ = "0.20.0-dev"
```